packages/block-editor/src/components/html-element-control/index.js

```diff
@@ -100,7 +100,7 @@ export default function HTMLElementControl( {
 			{ tagName === 'main' && hasMainElementElsewhere && (
 				<Notice status="warning" isDismissible={ false }>
 					{ __(
-						'Multiple <main> elements detected. This is not valid HTML and may cause accessibility issues. Please change this HTML element.'
+						'Multiple <main> elements detected. The duplicate may be in your content or in the page template. This is not valid HTML and may cause accessibility issues. Please change this HTML element.'
```

Comment:
```suggestion
						'Multiple <main> elements detected. The duplicate may be in your content or template. This is not valid HTML and may cause accessibility issues. Please change this HTML element.'
```

Suggestion: A minor adjustment.